build-tools/utils/files.js

```diff
@@ -2,21 +2,10 @@
 // SPDX-License-Identifier: Apache-2.0
 const fs = require('fs');
 const path = require('path');
-const prettier = require('prettier');
-
-const prettierConfigPath = path.join(process.cwd(), '.prettierrc');
-const prettierOptions = prettier.resolveConfig.sync(prettierConfigPath);
-
-function prettify(filepath, content) {
-  if (prettierOptions && ['.ts', '.js', '.json'].some(ext => filepath.endsWith(ext))) {
-    return prettier.format(content, { ...prettierOptions, filepath });
-  }
-  return content;
-}
 
 function writeFile(filepath, content) {
   fs.mkdirSync(path.dirname(filepath), { recursive: true });
-  fs.writeFileSync(filepath, prettify(filepath, content));
+  fs.writeFileSync(filepath, content);
```

Comment:
Prettier API is only asynchronous now. Since we depend on this `writeFile` in many places, migrating there does not fit into this PR so I removed that feature (for now)